### ISSUE TITLE
lite-vimeo-embed implementation

### ISSUE THREAD

**Issue by xplosionmind** [gfscott/eleventy-plugin-vimeo-embed#40]
As [eleventy-plugin-youtube-embed](https://github.com/gfscott/eleventy-plugin-youtube-embed) supports [lite-youtube-embed](https://github.com/paulirish/lite-youtube-embed), it would be awesome if this plugin could integrate [lite-vimeo-embed](https://github.com/luwes/lite-vimeo-embed) in the same way

**Comment by gfscott** [comment]
Makes sense! The lite-vimeo demo script is failing at the moment for some reason but should be doable.

<img width="1245" alt="Screen Shot 2022-03-28 at 7 54 28 PM" src="https://user-images.githubusercontent.com/547470/160506028-31aa1748-7c3c-458e-a367-dca8989fa920.png">
